src/main/java/org/testng/reporters/util/StackTraceTools.java

```diff
@@ -10,25 +10,30 @@
  * @version $Revision: 173 $
  */
 public class StackTraceTools {
+  private StackTraceTools() {
+    //defeat instantiation.
+  }
 
   /** Finds topmost position of the test method in the stack, or top of stack if <code>method</code> is not in it. */
   public static int getTestRoot(StackTraceElement[] stack,ITestNGMethod method) {
```

Comment:
Not part of the PR but a space is missing

Reply:
fixed